### PR TITLE
Multiline todo text feature added. 

### PR DESCRIPTION
**Pull request by arnabsen1729**
Now user can go to next line by `Shift+Enter` and can save the todo by `Enter` key

**Changes in Files:**
```
modified:   src/components/toDoList.js
modified:   src/components/todo.js
modified:   src/components/todoInput.js
```
**New Files:**
```
src/components/todoItem.js
```

I created a new component for each Todo Item, because that will help in further development.

- [x] Added code with proper linting
- [x] Added comments

**Things to keep in mind:**
- [ ] While developing the UI, the height of the input form needs to be considered, so that it resizes according to user input

https://github.com/shubham49/react-pwa/issues/12

### CODE REVIEW

File: src/components/todoInput.js [hunk 1]
```diff
@@ -2,36 +2,53 @@ import * as React from 'react';
...
-        props.onAdd(text);
+  const handleKeyDown = (event) => {
+    // handling 'Enter' keydown events
+    if (event.key === 'Enter') {
+      if (text.length === 0) {
         setText('');
+      } else if (event.shiftKey === false && text.length > 0) { 
+        onAddClick();
+      } else if (event.shiftKey === true) {
+        setText(text);
```
Comment (shubham49) [inline src/components/todoInput.js]: do we need this?

File: src/components/todoInput.js [hunk 2]
```diff
@@ -2,36 +2,53 @@ import * as React from 'react';
...
-        setText(event.target.value);
-
-
-    const onAddClick = () => {
-        props.onAdd(text);
+  const handleKeyDown = (event) => {
+    // handling 'Enter' keydown events
+    if (event.key === 'Enter') {
+      if (text.length === 0) {
         setText('');
```
Comment (shubham49) [inline src/components/todoInput.js]: do we need this?

Comment (arnabsen1729) [reply src/components/todoInput.js]: @shubham49  you are right. We don't need this, only one condition would suffice.

File: src/components/todoInput.js (hunk 1 above)
Comment (arnabsen1729) [reply src/components/todoInput.js]: @shubham49 this won't be necessary either. I just tested it. I am really sorry for making such mistakes :disappointed: I am making a new commit fixing these.

Comment (shubham49) [reply src/components/todoInput.js]: no issues

**Review by shubham49 — APPROVED** [review]

**Pull request merged**